packages/zipkin-instrumentation-request/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "zipkin-instrumentation-request",
+  "version": "0.6.2",
```

Comment:
I'm no lerna expert, but according to their docs, it will 

> Update the package.json of all updated packages to their new versions.
https://github.com/lerna/lerna#publish

I'm assuming this will publish `zipkin-instrumentation-request` as 0.6.2 since no version existed before.

```
zipkin-js $ lerna updated
Lerna v2.0.0-beta.38
Checking for updated packages...
Comparing with: v0.6.2

- zipkin-instrumentation-request
```